src/libais/ais8_200.cpp

```diff
@@ -42,6 +42,112 @@ Ais8_200_10::Ais8_200_10(const char *nmea_payload, const size_t pad)
   status = AIS_OK;
 }
 
+// ETA report
+Ais8_200_21::Ais8_200_21(const char *nmea_payload, const size_t pad)
+    : Ais8(nmea_payload, pad), eta_month(0), eta_day(0), eta_hour(0), eta_minute(0),
+      tugboats(0), air_draught(0.0) // TODO : add missing fields
+      {
+  assert(dac == 200);
+  assert(fi == 21);
+
+  if (num_bits != 248) {
+    status = AIS_ERR_BAD_BIT_COUNT;
+    return;
+  }
+
+  AisBitset bs;
+  const AIS_STATUS r = bs.ParseNmeaPayload(nmea_payload, pad);
+  if (r != AIS_OK) {
+    status = r;
+    return;
+  }
+
+  bs.SeekTo(88);
+  // UN country code                    12 bits
+  country = bs.ToString(88, 12);
+
+  // UN location code                   18 bits
+  location = bs.ToString(100, 18);
+  
+  // Fairway section number             30 bits
+  section = bs.ToString(118, 30);
+
+  // Terminal code                      30 bits
+  terminal = bs.ToString(148, 30);
+
+  // Fairway hectometre                 30 bits
+  hectometre = bs.ToString(178, 30);
+
+  // ETA at lock/bridge/terminal        20 bits 
+  eta_month  = bs.ToUnsignedInt(208, 4);
+  eta_day    = bs.ToUnsignedInt(212, 5);
+  eta_hour   = bs.ToUnsignedInt(217, 5);
+  eta_minute = bs.ToUnsignedInt(223, 6);
+
+  // Number of assisting tugboats        3 bits
+  tugboats = bs.ToUnsignedInt(229, 3);
```

Comment:
A comment for the unknown / defaults would be good.  e.g. `// 7: unknown`

Same for all other members that have special case values.